scripts/compile-kernel.sh

```diff
@@ -101,16 +114,16 @@ case "$COMPILER" in
 			--rm \
 			-v "$KERNEL_SRC_DIR:/linux/" \
 			kernel-gcc \
-			/bin/sh -c "cd linux && make clean && make $KERNEL_CONFIG && make -j32"
+			/bin/sh -c "cd linux && rm -f all.err && make clean && make $KERNEL_CONFIG && make -j32 $CHECKER"
```

Comment:
If the line is not getting too long (>100 characters), this line can stay as you propose in this commit.

Reply:
done